config/config.go

```diff
@@ -136,6 +136,10 @@ func Validate(cfg Config, opts InitOpts) error {
 		return fmt.Errorf("invalid `opts.MaxFileSize`; expected: >= %d, given: %d", minFileSize, opts.MaxFileSize)
 	}
 
+	if opts.ComputeBatchSize == 0 || opts.ComputeBatchSize%8 != 0 {
+		return fmt.Errorf("invalid `opts.ComputeBatchSize` expected: > 0 and divisible by 8, given: %d", opts.ComputeBatchSize)
```

Comment:
Why must a batch size be a multiple of 8?

NIT:
```suggestion
		return fmt.Errorf("invalid `opts.ComputeBatchSize` expected: > 0 and multiple of 8, given: %d", opts.ComputeBatchSize)
```

Reply:
Hi @poszu that is what the doc says for that parameter.

https://github.com/spacemeshos/post/blob/eee6c3c1f18ce80f80de1a4c9fea834dfbade8a2/config/config.go#L15

Also I'm not understanding your nit suggestion, the two messages seem equivalent.

Reply:
I think that comment is outdated. In the past it was assumed that a label can be 1-256 bits wide. Now it's fixed to 16 bytes. The output data will always be a multiple of a byte.

Regarding the nit. For me, it's easier to think in terms of something being a multiple of 8, that's it.

Reply:
Ok, so as long as compute batch size is greater than zero then it's valid?

Reply:
I think it's fine.